### DISCUSSION TITLE
Feature Request: Changing Column Width in lists and sublists

### DISCUSSION THREAD

**Discussion by ShotyMcFat** [burkybang/SuiteAdvanced-Feedback#218] [Ideas]
I'm not sure if you are able to do this with a Chrome Extension or if it has to be done in NetSuite itself

The Ability to Resize Column Width in lists and Sub lists, Similar to how you can in Excel.

**Comment by burkybang** [comment]
I had this feature in sublists, but I removed it earlier this year due to weird behavior. I had it remember column size across page navigation, and that's the part that didn't work too well sometimes. I could bring the feature back without the saving function if you think that would be useful.

**Comment by ShotyMcFat** [comment]
It would be Handy to have as an option, Are you Also able to add the ability to hide and Move the Columns by Drag and Drop.

If possible allow us to save manually (instead of it saving automatically like before), and have a return to default Button aswell.
